modules/administration-guide/pages/configuring-machine-autoscaling.adoc

```diff
@@ -0,0 +1,65 @@
+:_content-type: PROCEDURE
+:description: Configuring machine autoscaling
+:keywords: administration guide, machine, autoscaling, scaling
+:navtitle: Configuring machine autoscaling
+:page-aliases:
+
+[id="configuring-machine-autoscaling"]
+= Configuring machine autoscaling
+
+If a cluster has been configured to adjust the number of cluster nodes depending on resource needs, additional configuration is required to maintain the seamless operation of workspaces.
```

Comment:
I've got some bad news. One of us will have to edit this procedure to get rid of the passive voice and replace it with a more action-oriented phrasing (e.g. "additional configuration is required" x "you must add configuration") . It will make the text flow better.

Do you want to do it or should I?

Reply:
@deerskindoll , I see, I can make a new commit

Reply:
Updated to:
```
If you configured the cluster to adjust the number of nodes depending on resource needs,
you need additional configuration to maintain the seamless operation of {prod-short} workspaces.
```

Reply:
That looks great! But now you have to tackle the other instances of passive voice, too. That's why I wrote that one of us has to update the whole procedure.

Reply:
Sounds good, I've made another commit to remove the passive voice, but please feel free to make more changes where appropriate